app/presenters/topic_presenter.rb

```diff
@@ -7,7 +7,7 @@ def initialize(content_item)
   end
 
   def breadcrumbs
-    parent_breadcrumbs << topic_breadcrumb
```

Comment:
We can also remove the `topic_breadcrumb` function now

Reply:
removed function